src/Maker/MakeController.php

```diff
@@ -53,26 +54,42 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
 
     public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
     {
+        $this->projectDir = $command->getApplication()->getKernel()->getContainer()->getParameter('kernel.project_dir');
```

Comment:
Any better/canonical way to get the target dir? this currently makes the test suite fail, so needs to be resolve one way or another. Hint welcomed.

Reply:
`__construct(..., string $projectDir)`?